Benchmarks/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
Can we put "Benchmarks" in the existing "Performance" directory? QPS will stick around for sometime, as will the existing allocation tests so I'd rather keep them all in one place.